sdk/search/Microsoft.Azure.Search/CHANGELOG.md

```diff
@@ -1,103 +1,104 @@
-# GA SDK Releases
+GA SDK Releases
```

Comment:
Switch back to `#` syntax.

Reply:
Thanks. Fixed in this PR https://github.com/Azure/azure-sdk-for-net/pull/9168/files